### PR TITLE
Enable Terraform 1.9.* for CI tests

### PR DESCRIPTION
**Pull request by vmanilo**
Tests passes: https://github.com/vmanilo/terraform-provider-twingate/actions/runs/12243391198/job/34152892466

## Changes
- added terraform 1.9.*

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -126,7 +126,7 @@ jobs:
       matrix:
         terraform:
           - '1.8.*'
```
Comment (ekampf) [inline .github/workflows/ci.yml:128]: We can remove 1.8 as its EOL

Screenshot 2024-12-10 at 8 55 11 AM

File: .github/workflows/smoketests.yml [hunk 2]
```diff
@@ -46,7 +46,7 @@ jobs:
       matrix:
         terraform:
           - '1.8.*'
```
Comment (ekampf) [inline .github/workflows/smoketests.yml:48]: same

Comment (vmanilo) [reply .github/workflows/smoketests.yml]: sure, removed it

File: .github/workflows/ci.yml (hunk 1 above)
Comment (vmanilo) [reply .github/workflows/ci.yml]: good point

**Review by twingate-blee — APPROVED** [review]
lgtm

**Pull request merged**